schematic/models/GE_Helpers.py

```diff
@@ -119,6 +116,8 @@ def build_context(self):
             },
         }
 
+        anonymous_usage_statistics = AnonymizedUsageStatisticsConfig(enabled=False)
+
```

Comment:
This removes extra calls from leaving schematic.

Reply:
Should this be a comment?

Reply:
Added!